### ISSUE TITLE
Top level website

### ISSUE THREAD

**Issue by upkarlidder** [Prometeo-Platform/Prometeo#7]
This is the home page for Prometeo. How should this look like? This will show up on Github repo.
Current website: https://prometeoplatform.com/

Good examples:
1. [ClusterDuck](https://clusterduckprotocol.org/) 
2. [OpenEEW](https://openeew.com/)

FYI @krook @mrodrise

**Comment by Aquijada52** [comment]
I think for Pyrrah, going based of OpenEEW is best. Does IBM have a website creation application or how does one go about creating one for Pyrrah.

**Comment by TaeBlack** [comment]
i agree with alejandro, the homepage should simple but informative on the key points that the viewer should know. And since Pyrrah has so many visuals the OpenEEW example looked the cleanest implementing that.

**Comment by krook** [comment]
We don't have an IBM-specific process, but for OpenEEW we have a specific set of open source contributors that created a React Native implementation of the website based on a mockup from one of our designers. I think it makes sense to use that as the starting point: https://github.com/openeew/openeew.com

@TaeBlack @Aquijada52 do you have React experience?

**Comment by krook** [comment]
And we can also work with the LF on a new domain to host the site. They do that for openeew.com.

**Comment by krook** [comment]
Done, thanks everyone.
